sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/LogSensorTest.java

```diff
@@ -53,4 +66,10 @@ public void should_describe() {
     assertThat(sensorDescriptor.name()).isEqualTo(SHORT_LANG_NAME + " Analysis Log");
     assertThat(sensorDescriptor.languages()).isEmpty();     // should not filter per language
   }
+
+  @Test
+  public void executeLogsMessages() {
+    sensor.execute(context);
+    assertThat(logTester.logs()).hasSize(4);
```

Comment:
We could also assert the content of the logs, that would clarify why the size is 4

Reply:
Reading the correct values is the responsibility of `LogImporter`. `LogSensor` is just responsible for orchestrating it so in this test, I just need to be sure that something happens. I used `hasSize(4)` instead of `isNotEmpty()` to avoid some random noise in the log.